### ISSUE TITLE
TST: numpy.VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences is deprecated

### ISSUE THREAD

**Issue by pllim** [astropy/astropy#10183]
### Description

`numpy-dev` job on Travis CI is failing due to deprecation warning in `modeling` tests. That job is allowed to fail.

### Example log

https://travis-ci.org/github/astropy/astropy/jobs/677857013

```
==================================== ERRORS ====================================
_ ERROR collecting .tox/py37-test-devdeps/lib/python3.7/site-packages/astropy/modeling/tests/test_units_mapping.py _
../../.tox/py37-test-devdeps/lib/python3.7/site-packages/astropy/modeling/tests/test_units_mapping.py:51: in <module>
    @pytest.mark.parametrize(("model", "input_units", "return_units"), _models_with_units())
../../.tox/py37-test-devdeps/lib/python3.7/site-packages/astropy/modeling/tests/test_units_mapping.py:48: in _models_with_units
    return np.array([models, input_units, return_units]).T
E   numpy.VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
_ ERROR collecting .tox/py37-test-devdeps/lib/python3.7/site-packages/astropy/modeling/tests/test_units_mapping.py _
../../.tox/py37-test-devdeps/lib/python3.7/site-packages/astropy/modeling/tests/test_units_mapping.py:51: in <module>
    @pytest.mark.parametrize(("model", "input_units", "return_units"), _models_with_units())
../../.tox/py37-test-devdeps/lib/python3.7/site-packages/astropy/modeling/tests/test_units_mapping.py:48: in _models_with_units
    return np.array([models, input_units, return_units]).T
E   numpy.VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!
======================== 4 skipped, 2 errors in 23.96s =========================
ERROR: InvocationError for command /home/travis/build/astropy/astropy/.tox/py37-test-devdeps/bin/pytest --pyargs astropy /home/travis/build/astropy/astropy/docs --remote-data=any (exited with code 2)
```

**Comment by Waqasahmad0312** [comment]
how to fix    (numpy.VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences is deprecated) in python

**Comment by pllim** [comment]
This is already fixed in #10184 . Hope this helps!
